### ISSUE TITLE
All requests are synchronous

### ISSUE THREAD

**Issue by eliasmalik** [olot/tourism-app#6]
Asynchronous requests should be used, so that data is loaded as soon as the response comes in, but the rest of the code/functionality will still run and work. This leads to a snappier user experience, especially when you have lots of HTTP requests.

**Comment by sohilpandya** [comment]
@eliascodes you may want to link to the piece of code where this is happening, make it easier for the team to follow exactly where they are doing these reqs. 
https://github.com/olot/tourism-app/blob/master/js/tourism-app.js#L18-L23

**Comment by eliasmalik** [comment]
@sohilpandya yeah good point. Also on line 103 and 140 of the same file linked above
